model/db.js

```diff
@@ -78,6 +78,7 @@ const initDb = () => {
   runQuery(query, 'Initialised all tables!')
 }
 
+
```

Comment:
Could you remove this line? Not impt but it might cause merge conflict with a change im making on another branch